### PR TITLE
Nutrition

### PR DESCRIPTION
**Pull request by aspina7**
Just getting started on this - still long way to go

### CODE REVIEW

**Comment by zkamvar** [comment]
Alex, can you check out #133? It's begging to be merged :)

**Comment by aspina7** [comment]
Note to self: you have gotten to the bit about making weighted tables for indicators. However because you are a numpty you need to go back and change indicator vars in to single factor vars with levels, rather than splitting them in to a var for each category. 
This will make your life easier longterm. believe.

File: inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -782,8 +811,58 @@ The highest proportion of pregnant women was among the
...
+# paste these together seperated by a dash 
+hhrange <- paste0(
+  range(hhsize$n), 
+  collapse = "--")
+```
+
+
+
+The median number of households per cluster was
+`r clustermed`, with a range of `r clusterrange`. The median number of children
```
Comment (zkamvar) [inline inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd]: If you are going to be using these for inline results, then you really need to make sure that these are single length character vectors. `clustermed` here is a data frame, which causes weird things in the report.

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd [hunk 2]
```diff
@@ -478,20 +483,29 @@ study_data_cleaned <- study_data_cleaned %>%
...
 
-study_data_cleaned <- add_weights(study_data_cleaned, population_data_age, age_group, sex)
+study_data_cleaned <- add_weights(x = study_data_cleaned, 
+                                  p = population_data_age, 
+                                  age_group, sex)
 
 # drop unwanted rows  ---------------------------------------------------------
 
+# store the cases that you drop so you can describe them (e.g. non-consenting)
+dropped <- study_data_cleaned %>%
```
Comment (zkamvar) [inline inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: Wait... wasn't this done earlier?

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd [hunk 3]
```diff
@@ -655,16 +687,25 @@ year age group.
 ```{r cluster_hh_size}
 # get counts of the number of households per cluster
 clustersize <- study_data_cleaned %>% 
-  count(cluster)
+  group_by(cluster) %>% 
+  # return the number of unique household IDs by each cluster
+  summarise(
```
Comment (zkamvar) [inline inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: This form of exploding nested calls is a bit distracting. it's not difficult to read
`summarise(n = length(unique(q14_hh_no)))`

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd [hunk 4]
```diff
@@ -655,16 +687,25 @@ year age group.
...
+    n = length(
+      unique(q14_hh_no)
+      )
+    )
 
 # get the median number of households per cluster
 clustermed <- clustersize %>% 
-  pull(n) %>%
-  median()
+  summarise(
```
Comment (zkamvar) [inline inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: These are variables that are printed inline. They should be single strings or numbers. Here, you are making them data frames.

File: inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd (hunk 1 above)
Comment (aspina7) [reply inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd]: yeah i thought that but knitted and it came out fine....

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd (hunk 2 above)
Comment (aspina7) [reply inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: nope. Dropping non consents happens in the line after. The only previous back up is of study_data_raw

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd (hunk 3 above)
Comment (aspina7) [reply inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: okie dokie - just thought it was neater to seperate out when theres more than one thing going on. but am okay eitherway!

File: inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd (hunk 4 above)
Comment (aspina7) [reply inst/rmarkdown/templates/vaccination/skeleton/skeleton.Rmd]: ... dunno... worked when knitted...

**Pull request merged**